services/text-formatters.spec.js

```diff
@@ -42,14 +42,24 @@ describe('Text formatters', function() {
   test(metric, () => {
     given(999).expect('999')
     given(1000).expect('1k')
+    given(1100).expect('1.1k')
+    given(10100).expect('10k')
     given(999499).expect('999k')
     given(999500).expect('1M')
-    given(1578896212).expect('2G')
-    given(80000000000000).expect('80T')
+    given(1100000).expect('1.1M')
+    given(1578896212).expect('1.6G')
+    given(20000000000).expect('20G')
+    given(15788962120).expect('16G')
+    given(9949999999999).expect('9.9T')
+    given(9950000000001).expect('10T')
     given(4000000000000001).expect('4P')
+    given(4200000000000001).expect('4.2P')
+    given(7100700010058000200).expect('7.1E')
     given(71007000100580002000).expect('71E')
     given(1000000000000000000000).expect('1Z')
-    given(2222222222222222222222222).expect('2Y')
+    given(1100000000000000000000).expect('1.1Z')
+    given(2222222222222222222222222).expect('2.2Y')
+    given(22222222222222222222222222).expect('22Y')
```

Comment:
Can't wait til one of my packages has this many downloads 😄

Reply:
Share some links, I'm sure the maintainers will be happy to download them a few times and help towards that goal! 😁